extensions/PhabBugz/template/en/default/hook/attachment/edit-view.html.tmpl

```diff
@@ -0,0 +1,16 @@
+[%# This Source Code Form is subject to the terms of the Mozilla Public
+  # License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/.
+  #
+  # This Source Code Form is "Incompatible With Secondary Licenses", as
+  # defined by the Mozilla Public License, v. 2.0.
+  #%]
+
+[%
```

Comment:
Using an if block around the entire file is 30% faster than using RETURN,
because of exception handling in TT2.